### PR TITLE
Add documentation about "oc image mirror" commmand

### PR DESCRIPTION
**Pull request by legionus**
@bparees Please review.

### CODE REVIEW

File: install_config/registry/accessing_registry.adoc
Comment (bparees) [inline install_config/registry/accessing_registry.adoc]: i think i would have expected to find this under https://docs.openshift.org/latest/dev_guide/managing_images.html

(https://docs.openshift.org/latest/install_config/registry/accessing_registry.html#access-pushing-and-pulling-images should probably be moved there too, but that is part of a larger refactor that i think needs to happen w/ this content)

File: install_config/registry/accessing_registry.adoc
Comment (bparees) [inline install_config/registry/accessing_registry.adoc]: "you need to log in manually (using the `docker login` command) before executing the `oc image mirror` command."

File: install_config/registry/accessing_registry.adoc
Comment (bparees) [inline install_config/registry/accessing_registry.adoc]: s/to the S3 bucket/to an S3 bucket/

File: install_config/registry/accessing_registry.adoc
Comment (bparees) [inline install_config/registry/accessing_registry.adoc]: s/from S3/from the S3/

Comment (bparees) [reply install_config/registry/accessing_registry.adoc]: can you provide an example of the docker pull invocation you'd use to do that?

**Review by bparees — CHANGES_REQUESTED** [review]

File: install_config/registry/accessing_registry.adoc
Comment (bparees) [reply install_config/registry/accessing_registry.adoc]: where do the s3 credentials come from when doing this?

File: install_config/registry/accessing_registry.adoc
Comment (legionus) [reply install_config/registry/accessing_registry.adoc]: @bparees This has nothing to do with openshift images. This command is our clone of skopeo. You do not need to have imagestream to use this command. This is just client side general tool.

Comment (bparees) [reply install_config/registry/accessing_registry.adoc]: yes, but it also has nothing to do w/ the openshift registry (you can use it to mirror content between any two registries).

At least the images section is about managing images (not necessarily imagestreams).  Plus it's something a developer, not an administrator, would be doing.

Comment (legionus) [reply install_config/registry/accessing_registry.adoc]: @bparees OK. I will move this to managing_images.

**Comment by legionus** [comment]
@bparees Updated

File: install_config/registry/accessing_registry.adoc
Comment (legionus) [reply install_config/registry/accessing_registry.adoc]: @bparees I can't make it work with `minio` and `s3.amazonaws.com`. So, I can't give any examples here.

Comment (bparees) [reply install_config/registry/accessing_registry.adoc]: what is preventing you from making it work w/ s3?

File: dev_guide/managing_images.adoc
Comment (bparees) [inline dev_guide/managing_images.adoc]: "with a dot"

File: dev_guide/managing_images.adoc
Comment (bparees) [inline dev_guide/managing_images.adoc]: s/any/a/

File: dev_guide/managing_images.adoc
Comment (legionus) [reply dev_guide/managing_images.adoc]: @bparees Fixed

File: dev_guide/managing_images.adoc
Comment (legionus) [reply dev_guide/managing_images.adoc]: @bparees Fixed

File: dev_guide/managing_images.adoc
Comment (bparees) [inline dev_guide/managing_images.adoc]: per discussion w/ @smarterclayton about this and the issues you ran into with it, let's remove the s3-related bits of this doc.

Comment (legionus) [reply dev_guide/managing_images.adoc]: @bparees done

**Comment by bparees** [comment]
lgtm

@openshift/team-documentation ptal

**Comment by bmcelvee** [comment]
@legionus the content looks good. Once I'm able to merge this PR (when we have a 3.11 branch), I'll open a follow-up PR with some minor edits. 

In the meantime, @dongboyan77 PTAL. 

Thanks!

**Comment by bmcelvee** [comment]
@dongboyan77 PTAL

**Comment by bmcelvee** [comment]
@dongboyan77 PTAL :sun_with_face:

**Comment by dongboyan77** [comment]
LGTM, thanks

**Pull request merged**
